azkaban-web-server/src/web/js/azkaban/view/svg-graph.js

```diff
@@ -535,7 +545,7 @@ azkaban.SvgGraphView = Backbone.View.extend({
       var node = nodes[i];
       var gNode = node.gNode;
 
-      $(gNode).animate({"svgTransform": translateStr(node.x, node.y)}, 20);
+      $(gNode).animate({"svgTransform": translateStr(node.x, node.y)}, 250);
```

Comment:
Can we set this as a variable and make a comment where if it is changed it has UI side effects?